src/clr-addons/collapse-expand-section/collapse-expand-section.scss

```diff
@@ -0,0 +1,21 @@
+.ces-title {
+  color: #000;
```

Comment:
To support theming, we should not use hardcoded colors.
Is it possible to use h1-h6 tags to accomplish the same thing and ommit all this styling here?